di_website/error/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Go ahead and remove this file since we're not using it.

Reply:
Deleted admin.py